usaspending_api/broker/management/commands/update_transactions.py

```diff
@@ -0,0 +1,525 @@
+import logging
+import timeit
+from datetime import datetime
+
+from django.core.management.base import BaseCommand
+from django.db import connections, transaction as db_transaction, IntegrityError
+
+from usaspending_api.etl.broker_etl_helpers import dictfetchall
+from usaspending_api.awards.models import TransactionNormalized, TransactionFABS, TransactionFPDS
+from usaspending_api.awards.models import Award
+from usaspending_api.references.models import Agency, LegalEntity, SubtierAgency, ToptierAgency
+from usaspending_api.etl.management.load_base import copy, get_or_create_location, format_date, load_data_into_model
+from usaspending_api.etl.award_helpers import update_awards, update_contract_awards, update_award_categories
+import sys
+# start = timeit.default_timer()
+# function_call
+# end = timeit.default_timer()
+# time elapsed = str(end - start)
+
+
+logger = logging.getLogger('console')
+exception_logger = logging.getLogger("exceptions")
+
+# Lists to store for update_awards and update_contract_awards
+AWARD_UPDATE_ID_LIST = []
+AWARD_CONTRACT_UPDATE_ID_LIST = []
+
+subtier_agency_map = {subtier_agency['subtier_code']: subtier_agency['subtier_agency_id'] for subtier_agency in SubtierAgency.objects.values('subtier_code', 'subtier_agency_id')}
+subtier_to_agency_map = {agency['subtier_agency_id']: {'agency_id': agency['id'], 'toptier_agency_id': agency['toptier_agency_id']} for agency in Agency.objects.values('id', 'toptier_agency_id', 'subtier_agency_id')}
+toptier_agency_map = {toptier_agency['toptier_agency_id']: toptier_agency['cgac_code'] for toptier_agency in ToptierAgency.objects.values('toptier_agency_id', 'cgac_code')}
+
+class Command(BaseCommand):
+    help = "Checks what TASs are in Broker but not in Data Store"
+
+    @staticmethod
+    def update_transaction_assistance(db_cursor, fiscal_year=None, page=1, limit=500000):
```

Comment:
It looks like this is mostly just copied over from our current submission loader, right?